src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -638,6 +678,56 @@ void MenuGame::initInput()
             }
         },
         t::Once);
+    game.addInput(
+        {{k::F5}},
+        [this](ssvu::FT /*unused*/) { reloadLevelAssets(); },t::Once);
+}
+
+void MenuGame::reloadLevelAssets()
+{
+    if(state != States::SMain || !dialogText.empty() || !Config::getDebug()) return;
+    
+    noActions = 2;
+    assets.playSound("beep.ogg");
+
+    string reloadOutput = assets.reloadLevelData(levelData->packId, levelData->packPath, levelData->id);
+    if(reloadOutput != "invalid level folder path\n" && reloadOutput != "no matching level data file found\n")
```

Comment:
Hmm, it seems that here you are relying on the string output of `reloadLevelData` to decide what to do. This is brittle, because someone might change the return value of `reloadOutput` assuming that it is just used as a human-readable message, while you're actually using it programmatically in here. 

Generally, to observe multiple possibilities returned from a function, `enum class` is used in C++. If you need to carry some additional data per-possibility, you can use `std::variant` instead. Happy to give you some examples on Discord, if I am not misunderstanding the purpose of this code.

Reply:
I think you want both a `reloadOutput` string message for the human, and some sort of `reloadOutcome` enum/variant for the program here.